src/benchmarklib/benchmark_runner.cpp

```diff
@@ -177,6 +180,10 @@ void BenchmarkRunner::_benchmark_shuffled() {
   }
   _state.set_done();
 
+  for (auto& result : _results) {
```

Comment:
Maybe another comment or a pointer to the comment in benchmark_item_result.hpp might be helpful for the reader.

Reply:
laik dis?

Reply:
Naitz